dependencies/llvm/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ set(CMAKE_MAP_IMPORTED_CONFIG_RELWITHDEBINFO RelWithDebInfo Release MinSizeRel "
 set(CMAKE_MAP_IMPORTED_CONFIG_RELEASE Release MinSizeRel RelWithDebInfo "")
 
 find_package(LLVM ${Halide_REQUIRE_LLVM_VERSION} REQUIRED)
-find_package(Clang REQUIRED CONFIG HINTS "${LLVM_DIR}/../clang")
+find_package(Clang REQUIRED CONFIG HINTS "${LLVM_DIR}/../clang" "${LLVM_DIR}/../lib/cmake/clang")
```

Comment:
@alexreinking I don't remember why this change was made -- is it possible it's just a merge hickey?

Reply:
I have three system wide Ubuntu packages for LLVM installed. Needed this to get it to find the right Clang when I tested the PR.

Notice this is the same thing we do when finding LLD, so it was probably an oversight when I changed that.

Reply:
i.e. it's a desirable change, just a bit of a rider.